### ISSUE TITLE
Problems with JSON column

### ISSUE THREAD

**Issue by dalais** [64robots/nova-fields#82]
Hello guys!

After updating @Nova to 2.0.1, I have a resource that has popped up this error. What do you think is the problem?

```
"message": "Declaration of R64\\NovaFields\\Configurable::readOnly() should be compatible with Laravel\\Nova\\Fields\\Field::readonly($callback)",
    "exception": "ErrorException",
    "file": "/home/user/app/vendor/64robots/nova-fields/src/Boolean.php",
    "line": 7,
    "trace": [
        {
            "file": "/home/user/app/vendor/64robots/nova-fields/src/Boolean.php",
            "line": 7,
            "function": "handleError",
            "class": "Illuminate\\Foundation\\Bootstrap\\HandleExceptions",
            "type": "->"
        },
        {
            "file": "/home/user/app/vendor/composer/ClassLoader.php",
            "line": 444,
            "function": "include"
        },
        {
            "file": "/home/user/app/vendor/composer/ClassLoader.php",
            "line": 322,
            "function": "Composer\\Autoload\\includeFile"
        },
        {
            "function": "loadClass",
            "class": "Composer\\Autoload\\ClassLoader",
            "type": "->"
        },
        {
            "file": "/home/app/vezuchado/app/Nova/Resources/Order.php",
            "line": 79,
            "function": "spl_autoload_call"
        },
        ....
```

**Comment by disitec** [comment]
Yeah. Nova has added our readOnly method to their core and its signature was different to ours. We fixed that in 0.7.x version. Please upgrade and let me know if you have another issues.

**Comment by dalais** [comment]
Yes. Everything's normal. I updated to 0.7.3 and the problem disappeared. False alarm. Thank you!
